EnglishFilter/sections/antiadblock.txt

```diff
@@ -681,6 +681,8 @@ smallseotools.com#@#.adsbygoogle
 !
 !
 ! anti-adb
+! https://github.com/AdguardTeam/AdguardFilters/issues/118066
```

Comment:
Wrong place. Please move above early added rules.

Reply:
@Alex-302 Which row should it be in?